task-5/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from 'react';
+
+function App() {
+
+    var [fuel, setFeul] = useState(0);
+    var [alertText, setAlertTxt] = useState('Processing...');
+
+    useEffect(() => {
+      fetch('https://new.world.com/fleet/121')
+        .then(response => response.json())
+        .then(json => {
+          setFeul(json);
+          console.log(json);
+        })
+    });
+
+    useEffect(() => {
+      if (!fuel) setAlertTxt('Processing...'); else

```

Comment:
```suggestion
      if (!fuel) {
        setAlertTxt('Processing...');
      } else if (...) {
```
Avoid using if expressions without scope blocks. When you want to suddenly add more logic or log something out, you have to add and later remember to delete it to avoid random code changes, and that's not very practical. Also keep `else if` on the same line.